app/api/export/index.js

```diff
@@ -59,7 +59,7 @@ export default class ExportApi {
    *
    * No result will be returned. File is sent to user as side-effect.
    */
-  exportTransactions: ExportTransactionsRequest => Promise<ExportTransactionsResponse> = async (
+  exportTransactions: ExportTransactionsRequest => Promise<void> = async (
```

Comment:
```suggestion
  exportTransactions: ExportTransactionsRequest => Promise<ExportTransactionsResponse> = async (
```
pls don't remove this

Reply:
Sure, is this to keep the return types consistent, as in Request => Response types (because I see that the function really returns Promise<void>)